src/test/resources/db/scripts/RoleRepositoryIT_after.sql

```diff
@@ -0,0 +1,11 @@
+DELETE
+FROM invitation
+WHERE email IN ('RoleRepositoryIT-user-1@localhost', 'RoleRepositoryIT-user-2@localhost');
+
+DELETE
+FROM project
+WHERE name IN ('RoleRepositoryIT-project-1', 'RoleRepositoryIT-project-2');
+
+DELETE
+FROM jhi_user
+WHERE login IN ('RoleRepositoryIT-user-1', 'RoleRepositoryIT-user-2');
```

Comment:
probably not cleaning up after the tests? so we could look into the datasource on errors.